main.nf

```diff
@@ -24,72 +24,94 @@ def helpMessage() {
+      --transcript_fasta            Path to transcript fasta file
+      --splicesites                 Path to splice sites file for building HiSat2 index
       --gtf                         Path to GTF file
       --gff                         Path to GFF3 file
       --bed12                       Path to bed12 file
-      --saveReference               Save the generated reference files the the Results directory.
-      --saveTrimmed                 Save trimmed FastQ file intermediates
-      --saveAlignedIntermediates    Save the BAM files from the Aligment step  - not done by default
+      --saveReference               Save the generated reference files to the results directory
+      --gencode                     Use fc_group_features_type = 'gene_type' and pass '--gencode' flag to Salmon
 
-    Trimming options
+    Strandedness:
+      --forwardStranded             The library is forward stranded
+      --reverseStranded             The library is reverse stranded
+      --unStranded                  The default behaviour
+
+    Trimming:
+      --skipTrimming                Skip Trim Galore step
       --clip_r1 [int]               Instructs Trim Galore to remove bp from the 5' end of read 1 (or single-end reads)
       --clip_r2 [int]               Instructs Trim Galore to remove bp from the 5' end of read 2 (paired-end reads only)
       --three_prime_clip_r1 [int]   Instructs Trim Galore to remove bp from the 3' end of read 1 AFTER adapter/quality trimming has been performed
-      --three_prime_clip_r2 [int]   Instructs Trim Galore to re move bp from the 3' end of read 2 AFTER adapter/quality trimming has been performed
-
-    Presets:
-      --pico                        Sets trimming and standedness settings for the SMARTer Stranded Total RNA-Seq Kit - Pico Input kit. Equivalent to: --forward_stranded --clip_r1 3 --three_prime_clip_r2 3
-      --fcExtraAttributes           Define which extra parameters should also be included in featureCounts (default: gene_names)
-      --fcGroupFeatures             Define the attribute type used to group features. (default: 'gene_name')
-      --fcGroupFeaturesType         Define the type attribute used to group features based on the group attribute (default: 'gene_biotype')
+      --three_prime_clip_r2 [int]   Instructs Trim Galore to remove bp from the 3' end of read 2 AFTER adapter/quality trimming has been performed
+      --trim_nextseq [int]          Instructs Trim Galore to apply the --nextseq=X option, to trim based on quality after removing poly-G tails
+      --pico                        Sets trimming and standedness settings for the SMARTer Stranded Total RNA-Seq Kit - Pico Input kit. Equivalent to: --forwardStranded --clip_r1 3 --three_prime_clip_r2 3
+      --saveTrimmed                 Save trimmed FastQ file intermediates
 
-    Other options:
+    Ribosomal RNA removal:
+      --remove_rRNA                 Removes rRNA using SortMeRNA
```

Comment:
```suggestion
      --removeRiboRNA                 Removes ribosomal RNA using SortMeRNA
```

Reply:
Following on from discussion here:
https://github.com/nf-core/hic/issues/39

I think only these parameters need to be adjusted now to follow the `snake_case` and `camelCase` format.

Reply:
These will also need to be updated in `nextflow.config`, `parameters.settings.json`, `docs/` and in parameter summary section.

Reply:
Oh god why 😓